cmd/snap/cmd_set.go

```diff
@@ -116,6 +117,11 @@ func (x *cmdSet) Execute([]string) error {
 				// Not valid JSON-- just save the string as-is.
```

Comment:
No need for extra logic to generally validate the json, since it should be caught above.

Reply:
Awesome, agree, thank you!